### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by Ibtisam34**
- [ ]  In this project I  added the Views associated with the Models and Controllers I've already created for my Blog App.

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>Hi there</h1>
+<div class="all-users-container">
+<% @users.each do |user| %>
+    <%= link_to(user) do %>
+      <%= render "shared/user_details", user: user %>
```
Comment (danmainah) [inline app/views/users/index.html.erb]: - [ ] @Ibtisam34 kindly check the rendered file it doesn't exist in your shared folder ,kindly rectify to solve the error below 
image

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,15 @@
-<h1>welcome to the user page</h1>
+<div class="user-page-container">
+  <%= render "shared/user_details", user: @user %>
```
Comment (danmainah) [inline app/views/users/show.html.erb]: - [ ] @Ibtisam34 kindly change the rendered  link to match with the shared file naming, also in your design kindly ensure the posts section matches the width matches with the other page section 
image

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,20 @@
-<h1> posts are found here</h1>
+<div class="user-posts-page-container">
+ <%= render "shared/user_details", user: @user %>
+  <div class="post-comment-container">
+   <% @posts.each_with_index do |post| %>
+     <%= link_to user_post_path(user_id:@user.id, id: post.id), class:'nav-link' do%>
+       <%= render "shared/post_details", post: post %>
```
Comment (danmainah) [inline app/views/posts/index.html.erb]: - [ ] @Ibtisam34 kindly fix the links in lines 2 and 6 they are currently wrong making the app break, kindly also ensure your posts section aligns well with other section, also position of comments and likes counter
image

File: config/routes.rb [hunk 4]
```diff
@@ -5,4 +5,3 @@
   resources :posts, only: [:index, :show]
   end
 end
```
Comment (danmainah) [inline config/routes.rb:7]: - [ ] [optional] It would have been nice if your app had a root route, it makes it looks nices

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Ibtisam34 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
.

**Review by BregornOriginal — APPROVED** [review]
## Hi @Ibtisam34 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

IPhi

## Highlights

✅ Linters are passing
✅ Good work making changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**